<launch>
    <arg name="rasp_addr" default="160.69.69.111"/>
    <arg name="rasp_port" default="8888"/>
    <arg name="gpio_pin" default="2"/>

    <node pkg="pigpio_controller" type="pigpio_controller_node" name="pigpio_controller_node" output="screen">
        <param name="rasp_addr" type="str" value="$(arg rasp_addr)"/>
        <param name="rasp_port" type="str" value="$(arg rasp_port)"/>
        <param name="gpio_pin" type="int" value="$(arg gpio_pin)"/>
    </node>
</launch>